android/assets/jsons/Tutorials.json

```diff
@@ -1,619 +1,652 @@
-            {"text":"On the Tech screen, right-click or long press a technology to automatically queue it, even if this tech is in another branch than the item currently researched. Prerequisite techs to research will also be automatically queued."},
-            {"text":"Right-click or long press multiple techs to append them to the research queue, whatever their branch is."},
-            {},
-            {},
-            {"text":"Overview screens","header":3},
-            {},
-            {"text":"Reveal known resources on world screen","header":5,"color":"#fa0"},
-            {"text":"In the Resources overview, click on a resource icon to center the world screen on tiles already discovered and providing this resource."},
-            {"text":"Alternatively, click on the \"Unimproved\" number to center the world screen only on owned (by you or your allied City-states) tiles where the resource is not improved."},
-            {"text":"If more than one tile is available, click repeatedly on the notification to cycle through all of them."},
-            {},
-            {"text":"Show diagram line colors","header":5,"color":"#fa0"},
-            {"text":"In Politics overview > Show diagram, you can click anywhere inside the diagram to display a table listing all relationship line colors and their meaning."},
-            {},
-            {},
-            {"text":"Miscellaneous","header":3},
-            {},
-            {"text":"External links","header":5,"color":"#fa0"},
-            {"text":"External links support right-click or long press to copy the link to the clipboard instead of launching the browser."},
-            {"text":"Example: The 'Open Github page' button on the Mod management screen."}
-        ]
-    }
-]
+	// Each entry is a tutorial, but the tutorial may be spread over separate paragraphs.
+  
+	  {
+		  "name": "Civilopedia",
+		  "civilopediaText": [
+			  {"text":"Welcome to the Civilopedia!","header":4,"color": "#fa0"},
+			  {"text":"Here you can find information - general help, rules, and what makes up the game world."},
+			  {},
+			  {"text":"How to find information","header":4},
+			  {"text":"Select categories with the buttons on top of the screen. Also up there is the button to leave Civilopedia and go back to where you were before.","starred":true,"color":"#666"},
+			  {"text":"Each category has a list of entries on the left of the screen, sorted alphabetically (with few exceptions). Clicking an entry will update the center pane were you are currently reading this.","starred":true,"color":"#666"},
+			  {"text":"Lines can link to other Civilopedia entries, they are marked with a chain link symbol like this one. You can click anywhere on the line to follow the link.", "link":"Tutorial/New Game"},
+			  {"text":"The current category is special - all articles on general concepts are here. It is called 'Tutorials' because you can revisit these here, too.","starred":true,"color":"#666"},
```

Comment:
```suggestion
			  {"text":"The current category is special - all articles on general concepts are here. It is called \"Tutorials\" because you can revisit these here, too.","starred":true,"color":"#666"},
```